src/pages/index.js

```diff
@@ -278,6 +287,20 @@ const Index = ({ data: initialData }) => {
   const userGlobalId = get(user, 'id')
   const [tabId] = useState(uuid())
 
+  // this is a temporary workaround as the latest updates to the
+  // relay store do not push into this component, so we are manually
+  // toggling state and a rerender when we successfully fire the
+  // SetHasViewedIntroFlowMutation
+  const [hasViewedIntroFlow, setHasViewedIntroFlow] = useState(
+    get(user, 'hasViewedIntroFlow')
+  )
+  useEffect(() => {
+    setHasViewedIntroFlow((prevState) =>
+      user.hasViewedIntroFlow !== prevState
+        ? user.hasViewedIntroFlow
+        : prevState
+    )
+  }, [user])
```

Comment:
I'm getting an error on the first view of this after login. I believe it's because the user data might not exist (uncached), and moving this down below the `if(!data) { ... }` should fix it (?).